docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/detection/roi-align-rotated-14.rst

```diff
@@ -0,0 +1,123 @@
+.. {#openvino_docs_ops_detection_ROIAlignRotated_14}
+ROIAlignRotated
+===============
+
+
+.. meta::
+  :description: Learn about ROIAlignRotated-14 - an object detection operation, 
+                which can be performed on three required input tensors.
+
+
+**Versioned name**: *ROIAlignRotated-14*
+
+**Category**: *Object detection*
+
+**Short description**: *ROIAlignRotated* is a *pooling layer* used over feature maps of non-uniform input sizes and outputs a feature map of a fixed size.
+
+**Detailed description**: `Reference <https://arxiv.org/abs/1703.06870>`__.
```

Comment:
As the mentioned paper describes the basic ROIAlign, worth to search for a paper related to the "rotated" ROI Align, relevant to be referenced as well.

Reply:
I haven't found any related paper to ROIAlignRotated - it is just a simple extension of ROIAlign by adding affine transformation to sampling points.

Reply:
BTW, if there is a whole paper about ROIAlignRotated somewhere in the net, I can immediately write similar paper about extending ROIAlign with perspective(camera) transformation - not sure why they didn't do it in the first place ;)

Reply:
I did a little digging and found https://arxiv.org/abs/1703.01086 (in there is also implementation on GH), this one seem to be one of earliest papers on topic that I was able to found

Reply:
Sure, but I did not used that paper to implement op, it looks loosely related to the operator itself, but I can added as 'reference'. Hope it will not be misleading to the people. It is up to you.